pkgs/test/lib/src/executable.dart

```diff
@@ -2,178 +2,22 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// TODO(nweiz): This is under lib so that it can be used by the unittest dummy
-// package. Once that package is no longer being updated, move this back into
-// bin.
-import 'dart:async';
-import 'dart:io';
-
-import 'package:async/async.dart';
-import 'package:path/path.dart' as p;
-import 'package:source_span/source_span.dart';
-import 'package:stack_trace/stack_trace.dart';
-import 'package:test_api/src/utils.dart'; // ignore: implementation_imports
-
-import 'runner.dart';
-import 'runner/application_exception.dart';
-import 'runner/configuration.dart';
-import 'runner/version.dart';
-
-import 'util/exit_codes.dart' as exit_codes;
-import 'util/io.dart';
-
-/// A merged stream of all signals that tell the test runner to shut down
-/// gracefully.
-///
-/// Signals will only be captured as long as this has an active subscription.
-/// Otherwise, they'll be handled by Dart's default signal handler, which
-/// terminates the program immediately.
-final _signals = Platform.isWindows
-    ? ProcessSignal.sigint.watch()
-    : StreamGroup.merge(
-        [ProcessSignal.sigterm.watch(), ProcessSignal.sigint.watch()]);
-
-/// Returns the path to the global test configuration file.
-final String _globalConfigPath = () {
-  if (Platform.environment.containsKey('DART_TEST_CONFIG')) {
-    return Platform.environment['DART_TEST_CONFIG'];
-  } else if (Platform.operatingSystem == 'windows') {
-    return p.join(Platform.environment['LOCALAPPDATA'], 'DartTest.yaml');
-  } else {
-    return '${Platform.environment['HOME']}/.dart_test.yaml';
-  }
-}();
+// ignore: implementation_imports
+import 'package:test_core/src/executable.dart' as executable;
```

Comment:
Yes, flutter_test and flutter_tools will continue to use src imports. This is expected since the API of test_core and test_api has not stabilized. Once we are comfortable with the touch points we will promote them to public and update Flutter, Google3, Bolt etc.